content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -22,6 +22,100 @@ Configuring the [aggregation layer](/docs/concepts/extend-kubernetes/api-extensi
 There are a few setup requirements for getting the aggregation layer working in your environment to support mutual TLS auth between the proxy and extension apiservers. Kubernetes and the kube-apiserver have multiple CAs, so make sure that the proxy is signed by the aggregation layer CA and not by something else, like the master CA.
 {{< /note >}}
 
+{{% /capture %}}
+
+{{% capture authflow %}}
+
+## Authentication Flow
+
+Unlike Custom Resource Definitions (CRDs), the Aggregation API involves another server - your Extension API Server - in addition to the standard Kubernetes apiserver. The Kubernetes apiserver will need to communicate with your extension server, and your extension server will need to communicate with the Kubernetes apiserver. In order for this communication to be secured, the apiserver uses x509 certificates to authenticate itself to the extension server.
+
+This section describes how the authentication and authorization flows work, and how to configure them.
+
+The flow can be seen in [this swimlane diagram](https://swimlanes.io/d/mF9PizcZX).
```

Comment:
can we pull the resulting diagram into this repo (along with the text source that generated it)

Reply:
I pulled the diagram in as a `.png`, and put the swimlanns source in a comment in the source file, so we have it, without it messing up the document itself.